### ISSUE TITLE
Request: 'any trash'

### ISSUE THREAD

**Issue by Keverjeroen** [apstemmer/com.athom.trashchecker#11]
Hi,

Thanks for the great app! Is it posible to add in the card an option 'any trash'. Now I need to make a flow for every trash-type. But I want to make just 1 flow for all trash-types (rest, gft, papier, etc).

Thanks!

**Comment by robertraaijmakers** [comment]
I think that with the new flow options this is possible when there comes the option to make an "or" column.  Then you can drag all cards in the or and have the same effect.

**Comment by spoelly** [comment]
would be nice if this worked. But i have a flow with a notification which trash i have to put outside. Since there is no tag for the message i like to sent i can't get this in one flow with the "or" option.

**Comment by robertraaijmakers** [comment]
Yes, it would indeed. Trying to implement something like it with a global tag and the settings page. Will see if I can release a beta version soon.
